diff --git a/dist/runtime/parser/handlers/code.js b/dist/runtime/parser/handlers/code.js
index f87404ebde2bd152e393be9c8474e53ad3629cae..735081241215bf6246946ec7c796c794b02996c1 100644
--- a/dist/runtime/parser/handlers/code.js
+++ b/dist/runtime/parser/handlers/code.js
@@ -1,9 +1,8 @@
-import { detab } from "detab";
 import { parseThematicBlock } from "./utils.js";
 export default (state, node) => {
   const lang = (node.lang || "") + " " + (node.meta || "");
   const { language = "text", highlights, filename, meta } = parseThematicBlock(lang);
-  const value = node.value ? detab(node.value + "\n") : "";
+  const value = node.value ? node.value + "\n" : "";
   let result = {
     type: "element",
     tagName: "code",
diff --git a/dist/runtime/parser/handlers/inlineCode.js b/dist/runtime/parser/handlers/inlineCode.js
index a79488ac2d4b997ee1dcc244db1ea6103a223639..f2847359799aad0d47b8d57396fec97c05dc237b 100644
--- a/dist/runtime/parser/handlers/inlineCode.js
+++ b/dist/runtime/parser/handlers/inlineCode.js
@@ -5,7 +5,7 @@ export default function inlineCode(state, node) {
   const result = {
     type: "element",
     tagName: "code",
-    properties: node.attributes || {},
+    properties: { ...node.attributes, code: text.value },
     children: [text]
   };
   const classes = (result.properties.class || "").split(" ");
