diff --git a/lib/server/registry/index.js b/lib/server/registry/index.js
index 9a771c11b992a026552028ab5bb2ffcc9789d848..c7a8e0a8fb64e0fef783c97c29ab3370960b2ccb 100644
--- a/lib/server/registry/index.js
+++ b/lib/server/registry/index.js
@@ -476,7 +476,7 @@ const registryDirectory = (() => {
     result = envDefined;
   } else {
     let cacheDirectory;
-    if (process.platform === "linux")
+    if (process.platform === "linux" || process.platform === "android")
       cacheDirectory = process.env.XDG_CACHE_HOME || import_path.default.join(import_os.default.homedir(), ".cache");
     else if (process.platform === "darwin")
       cacheDirectory = import_path.default.join(import_os.default.homedir(), "Library", "Caches");
