diff --git a/lib/server/registry/index.js b/lib/server/registry/index.js
index db598f19493c257070e5889be80c1dafd09c4d75..60b63ebce0f385009294b588f4d36efe2d00cc82 100644
--- a/lib/server/registry/index.js
+++ b/lib/server/registry/index.js
@@ -431,7 +431,7 @@ const registryDirectory = (() => {
     result = envDefined;
   } else {
     let cacheDirectory;
-    if (process.platform === "linux")
+    if (process.platform === "linux" || process.platform === "android")
       cacheDirectory = process.env.XDG_CACHE_HOME || import_path.default.join(import_os.default.homedir(), ".cache");
     else if (process.platform === "darwin")
       cacheDirectory = import_path.default.join(import_os.default.homedir(), "Library", "Caches");
