diff --git a/dist/vue-tippy.d.ts b/dist/vue-tippy.d.ts
index 617ffdf69d5d8b7de07de313eb84a07b53c3e08f..ed94c756448e8b71fbba28c3f8a5c150dab2aff9 100644
--- a/dist/vue-tippy.d.ts
+++ b/dist/vue-tippy.d.ts
@@ -10,6 +10,7 @@ import { Props } from 'tippy.js';
 import { PropType } from 'vue';
 import { Ref } from 'vue';
 import { roundArrow } from 'tippy.js';
+import { ShallowRef } from 'vue';
 import tippy from 'tippy.js';
 import { UnwrapNestedRefs } from 'vue';
 import { VNode } from 'vue';
@@ -718,7 +719,7 @@ export declare function useSingleton(instances: TippyInstances, optionalProps?:
     singleton: import("vue").Ref<import("tippy.js").CreateSingletonInstance<CreateSingletonProps<Props>> | undefined>;
 };
 
-export declare function useTippy(el: Element | (() => Element) | Ref<Element> | Ref<Element | undefined>, opts?: TippyOptions, settings?: {
+export declare function useTippy(el: Element | (() => Element) | Ref<Element> | Ref<Element | undefined> | ShallowRef<Element | null>, opts?: TippyOptions, settings?: {
     mount: boolean;
     appName: string;
 }): {
